libr/core/disasm.c

```diff
@@ -869,14 +870,15 @@ static int var_comparator(const RAnalVar *a, const RAnalVar *b){
 	return false;
 }
 
-static void handle_show_functions(RCore *core, RDisasmState *ds) {
+static void ds_show_functions(RDisasmState *ds) {
 	RAnalFunction *f;
+	RCore *core = ds->core;
 	bool demangle, call;
 	const char *lang;
 	char *fcn_name;
 	char *sign;
 
-	if (!core || !ds || !ds->show_functions) {
+	if (!ds->show_functions) {
```

Comment:
No it's ok! :) The important thing is that it's removed for a right reason! Thanks